code/game/objects/items/stacks/medical.dm

```diff
@@ -127,6 +127,21 @@
 	self_delay = 20
 	max_amount = 12
 
+/obj/item/stack/medical/gauze/wirecutter_act(mob/living/user, obj/item/I)
+	if(get_amount() < 2)
+		to_chat(user, "<span class='warning'>You need at least two gauze to do this!</span>")
+		return
+	var/obj/item/stack/sheet/cloth/new_item = new(usr.loc)
+	user.visible_message("[user] cut [src] into pieces of cloth with [I].", \

```

Comment:
I think the verb should be `cuts` here but other than that looks good. 2 gauze into 1 cloth, right?